### ISSUE TITLE
"Please rename transcoding.webtorrent.enabled key to transcoding.web_videos.enabled in your configuration file"

### ISSUE THREAD

**Issue by Jordibgzashtita** [Chocobozzz/PeerTube#6093]
### Describe the current behavior

WHERE IS.  looking in entire peertube directory  transcoding.webtorrent.enabled is NOT FOUND. then how it is supposed to be found. i cloned master git clone to seek where it is AND IT WASN'T THERE TOO. after 6.01 update from 5.x now peertube complaint: "stack": "Error: Please rename transcoding.webtorrent.enabled key to transcoding.web_videos.enabled in your configuration file\n    at checkRemovedConfigKeys (file:///var/www/peertube/versions/peertube-v6.0.1/dist/core/initializers/checker-after-init.js:82:19)\n    at checkConfig (file:///var/www/peertube/versions/peertube-v6.0.1/dist/core/initializers/checker-after-init.js:30:5)\n    at file:///var/www/peertube/versions/peertube-v6.0.1/dist/server.js:27:5",
 BUT WHERE i look in config directory too and it isnt there too. 

### Steps to reproduce

1.
2.
3.

### Additional information

* PeerTube instance:
  * URL:
  * Version:
  * NodeJS version:
  * Ffmpeg version:

* Browser name, version and platforms on which you could reproduce the bug:
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

**Comment by Chocobozzz** [comment]
Hello,

Please paste the output of `sudo grep -r webtorrent /var/www/peertube/config`

**Comment by CryptoSiD** [comment]
I have the exact same issue.

Here's the output of "sudo grep -r webtorrent /var/www/peertube/config"

/var/www/peertube/config/production.yaml.new:  # Same settings but for webtorrent videos
/var/www/peertube/config/production.yaml.new:  webtorrent:
/var/www/peertube/config/production.yaml.new:  # If you also enabled the webtorrent format, it will multiply videos storage by 2
/var/www/peertube/config/production.yaml:  # Same settings but for webtorrent videos
/var/www/peertube/config/production.yaml:  webtorrent:
/var/www/peertube/config/production.yaml:  # If you also enabled the webtorrent format, it will multiply videos storage by 2

**Comment by Chocobozzz** [comment]
@CryptoSiD Please follow IMPORTANT NOTES Of https://github.com/Chocobozzz/PeerTube/releases/tag/v6.0.0

In particular, the following section:

> Configuration key that you must update in your production.yaml if not automatically done by your upgrade script:

**Comment by CryptoSiD** [comment]
> Configuration key that you must update in your production.yam

I don't have "storage.videos", nor "transcoding.webtorrent" neither "bject_storage.videos".

I don't know how to fix the problem.

**Comment by Chocobozzz** [comment]
`storage.videos` is the key below `storage:` and then `videos` so for example https://github.com/Chocobozzz/PeerTube/blob/release/5.2.0/config/production.yaml.example#L131

Same for `transcoding.webtorrent`: search for `transcoding:` key then rename the subkey `webtorrent`

**Comment by CryptoSiD** [comment]
Thank you :)

**Comment by Chocobozzz** [comment]
@aukfood directory is `/your-peertube-base-directory/config` and not `/your-peertube-base-directory/peertube-latest/config` :)

**Comment by Jordibgzashtita** [comment]
Hi guys sorry for not update. I fixed it by comparing original and actual file. It has missing, names. So after I add them peertube start without problems. 


2. There is problem I. Latest version. Maybe from my side, I see that adding videos sometimes they don't appear, only show audio transcode. I put only 4xx mode p. To safe space when safe videos. if enable 720 p it will use more space, yes correct if there is text people's can see it. But much hard disk space. Now problem came how we say, when try upload video or YouTube clone video, sometimes video don't appear. Only audio hear. Can someone help me. Here is one case: 

https://peertube.bgzashtita.es/w/eZVj1KPYc1zT4UNh2dWYxP


Second case:

https://peertube.bgzashtita.es/w/vcW84Er3xcpGVceKpYsyvG


3. Plugin voice to text, when they install  the old version it fail with error. I try install other voice to text and same error.... In past we found manual how to delete something.... And i don't remember what steps to do it.... Can someone wrote good manual how to correctly install peertube from scratch to professional on latest Ubuntu. Or what Ubuntu Version is much better. Now if I remember it use nodejd ? can someone write manual how in  install latest stable. And in case it is unsupported in peertube how to downgrade or install latest stable. In past when peertube was working and try convert audio to text, something star ge was happened. It was translating then after maybe 30 min. It just stop. Can someone make audio to text and if possible translate it from language to other. 


We remember, when in start version of peerube, after update, it show original with actual version of peertube or nginx config. When it show changes it show what was new. in if I remember on latest peertube it didn't work. For that I didn't know what was missing.
